curvefs/src/mds/fs_manager.cpp

```diff
@@ -884,6 +896,31 @@ void FsManager::GetAllFsInfo(
     LOG(INFO) << "get all fsinfo.";
 }
 
+FSStatusCode FsManager::UpdateFsUsedBytes(
+    const std::string& fsName, int64_t deltaBytes) {
+    WriteLockGuard wlock(fsUsageMutex_);
+
+    FsUsage usage;
+    auto ret = fsStorage_->GetFsUsage(fsName, &usage, true);
+    if (ret == FSStatusCode::NOT_FOUND) {
+        usage.set_usedbytes(deltaBytes);
+        return fsStorage_->SetFsUsage(fsName, usage);
+    }
+
+    if (ret != FSStatusCode::OK) {
+        LOG(WARNING) << "UpdateFsUsedBytes fail, get fs usage fail, fsName = "
+                     << fsName << ", errCode = " << FSStatusCode_Name(ret);
+        return ret;
+    }
+
+    if (deltaBytes < 0 && usage.usedbytes() < -deltaBytes) {
+        usage.set_usedbytes(0);
+    } else {
+        usage.set_usedbytes(usage.usedbytes() + deltaBytes);
+    }
```

Comment:
```suggestion
    usage.set_usedbytes(std::max(0, usage.usedbytes() + deltaBytes));
```

Are they equivalent?

Reply:
we need deal with uint64_t + int64_t